### DISCUSSION TITLE
`/api/auth/signin` looks terrible

### DISCUSSION THREAD

**Discussion by corysimmons** [nextauthjs/next-auth#4916] [Ideas]
Would you accept a PR?

**Comment by adaptive-shield-matrix** [comment]
like https://github.com/TimMikeladze/next-auth-mui ?

**Reply by corysimmons** [reply]
That also looks terrible.

Something like this:
- https://tailwindui.com/components/application-ui/forms/sign-in-forms
- https://ui.supabase.io/components/auth

Possibly even extend next-auth's config to have themes and just use the current one as the default so this change wouldn't be a breaking change.

**Comment by balazsorban44** [comment]
Luckily you can customize it! :) https://next-auth.js.org/configuration/pages#sign-in-page

**Reply by corysimmons** [reply]
True but I'm pretty lazy and would love to just set some config option like `theme: 'tailwindy'` or something if you'd be interested in a PR. 😬

**Comment by reachkrishnaraj** [comment]
following
